splight_lib/models/dashboard.py

```diff
@@ -66,7 +66,7 @@ class ChartItem(BaseModel):
     ref_id: str
     type: ChartItemType = ChartItemType.QUERY
     label: str
-    hidden: bool = False
+    hidden: bool | None = None
```

Comment:
Is this the default value in the platform?

Reply:
I've seen many in production with this field nulled